### PR TITLE
revise some expressions of shapshot, restore, and clone

### PR DESCRIPTION
**Pull request by zhongdeyu**
My first PR as a test

### CODE REVIEW

File: docs/volume-operations/snapshot-restore-and-clone.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-title: Snapshot, Restore And Clone
-sidebar_label: Snapshot, Restore And Clone
+title: Snapshot, Restore and Clone
+sidebar_label: Snapshot, Restore and Clone
 ---
 
 ## Snapshot
 
-IOMesh provides the ability to create a snapshot of an existing persistent volume.
+IOMesh is able to create a snapshot for an existing persistent volume (PV) by using a VolumeSnapshot object.
```
Comment (nanana-w) [inline docs/volume-operations/snapshot-restore-and-clone.md:9]: > IOMesh is able to create a snapshot for an existing persistent volume (PV) by using a VolumeSnapshot object.

> User can restore a volume snapshot by creating a PVC in which the `dataSource` field references to a snapshot.

> Users can clone a persistent volume (PV) by creating a PVC while adding a dataSource linked to an existing PVC in the same namespace.

像这类型的，我们需不需要统一一下，使用同一种句式呢？

File: docs/volume-operations/snapshot-restore-and-clone.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
-A VolumeSnapshot object defines a request of taking a snapshot of the PVC.
+A VolumeSnapshot object defines a request for taking a snapshot by using a PVC.
```
Comment (kylezh) [inline docs/volume-operations/snapshot-restore-and-clone.md:11]: It should be `snapshot of the PVC`

File: docs/volume-operations/snapshot-restore-and-clone.md (hunk 1 above)
Comment (zhongdeyu) [reply docs/volume-operations/snapshot-restore-and-clone.md:9]: 嗯，统一一下比较好些，第一句`IOMesh is able to create ... ` 用 `IOMesh`做主语不很合适，可以改为类似这样的 `Users can create snapshots for an existing ...`

File: docs/volume-operations/snapshot-restore-and-clone.md (hunk 2 above)
Comment (zhongdeyu) [reply docs/volume-operations/snapshot-restore-and-clone.md:11]: 好的再改回来，我修改时理解不准确

File: docs/volume-operations/snapshot-restore-and-clone.md [hunk 3]
```diff
@@ -102,9 +102,9 @@ spec:
...
 
-After applying it, a clone of `existing-pvc` will be created.
+After applying the YAML file, a clone of `existing-pvc` will be created.
 
 There are some limitations on clone operation:
 
-1. A cloned PVC must exist at the same namespace with the original PVC with same StorageClass.
-2. The new and source PVC must have the same VolumeMode setting.
+1. A cloned PVC must exist in the same namespace as the original PVC with the same StorageClass.
+2. The new source PVC must have the same VolumeMode setting.
```
[the remark this answers is not in the document]
Comment (zhongdeyu) [reply docs/volume-operations/snapshot-restore-and-clone.md:110]: 好的，感觉这两句的信息有重复，可以合并一下，比如：
`A cloned PVC must exist in the same namespace as the original PVC and must have the same StorageClass. Both PVCs must have the same VolumeMode setting.`
or
`A cloned PVC must exist in the same namespace as the original PVC. Both PVCs must have the same StorageClass and VolumeMode settings.` (`setting`是否有可能为复数？)    
看看这样理解是否正确：源PVC和克隆PVC要求有三个相同属性，`namespace`, `StorageClass`, `VolumeMode setting`

File: docs/volume-operations/snapshot-restore-and-clone.md (hunk 1 above)
Comment (nanana-w) [reply docs/volume-operations/snapshot-restore-and-clone.md:9]: 嗯嗯 我觉得可以

File: docs/volume-operations/snapshot-restore-and-clone.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nanana-w) [reply docs/volume-operations/snapshot-restore-and-clone.md:110]: 正确的

File: docs/volume-operations/snapshot-restore-and-clone.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ example-snapshot   mongodb-data-pvc     6Gi            snapcontent-fb64d696-725b
 
 ## Restore
 
-User can restore volume snapshots by creating a PVC which `dataSource` field reference to a snapshot.
+User can restore a volume snapshot by creating a PVC in which the `dataSource` field references to a snapshot.
```
Comment (zhongdeyu) [inline docs/volume-operations/snapshot-restore-and-clone.md:50]: 对于这句话，我的理解如下：
“通过创建一个PVC，来恢复一个（或多个? 复数?） volume snapshots，而这个创建出来的PVC的dataSource域，参考指向一个snapshot（单数）。“
前后文似乎是在说一个snapshot。不知上述理解是否正确？

Comment (nanana-w) [reply docs/volume-operations/snapshot-restore-and-clone.md:50]: 我的理解是：通过创建一个 PVC，数据源是一个快照，以此来实现从快照恢复一个卷

Comment (nanana-w) [reply docs/volume-operations/snapshot-restore-and-clone.md:50]: 另外，这里是否也修改为 `users `

Comment (zhongdeyu) [reply docs/volume-operations/snapshot-restore-and-clone.md:50]: > 另外，这里是否也修改为 `users `
嗯嗯，应该复数比较好

**Comment by nanana-w** [comment]
LGTM
请 @kylezh 确认

**Review by kylezh — APPROVED** [review]

**Pull request merged**